resources-generator/build.gradle.kts

```diff
@@ -46,7 +46,7 @@ kotlin {
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile>()
     .configureEach {
         compilerOptions.jvmTarget.set(JvmTarget.JVM_11)
-        compilerOptions.languageVersion.set(KotlinVersion.KOTLIN_1_7)
+        compilerOptions.languageVersion.set(KotlinVersion.KOTLIN_2_0)
```

Comment:
1.9 and older are deprecated and fail to sync/build. This is a pretty big breaking change.